### PR TITLE
created homepage markup

### PR DESCRIPTION
**Pull request by Admiral-kalina**
check pls

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import "../src/styles/index.css"
```
Comment (HeibaiL) [inline .storybook/preview.js]: put it below absolute import

File: src/__stories__/Home.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+//components
+import HomePage from "@/pages/index";
+import Layout from "@/layouts/Layout";
```
Comment (HeibaiL) [inline src/__stories__/Home.stories.tsx:6]: create a folder "mainLayout" and create index file for layout

File: src/__stories__/components/Header.stories.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { ComponentStory, ComponentMeta } from "@storybook/react";
 
 //components
-import HeaderComponent from "@/components/Header";
+import HeaderComponent from "@/layouts/Header";
```
Comment (HeibaiL) [inline src/__stories__/components/Header.stories.tsx]: why header is in layout? put it inside of components folder

File: src/assets/icons/CallImg.jsx [hunk 4]
```diff
@@ -0,0 +1,518 @@
+import React from "react";
```
Comment (HeibaiL) [inline src/assets/icons/CallImg.jsx]: removve it. Add png instead of svg for large images

File: src/assets/icons/LaptopImg.jsx [hunk 5]
```diff
@@ -0,0 +1,561 @@
+import React from "react";
```
Comment (HeibaiL) [inline src/assets/icons/LaptopImg.jsx]: SAME HERE

File: src/layouts/Header/Header.tsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  const { isMobile } = useWindowDimension();
+
+  const renderNavigationList = () => (
+    <ul className={styles.navList}>
+      <li className={styles.navItem}><a className={styles.navLink} href="#">About</a></li>
+      <li className={styles.navItem}><a className={styles.navLink} href="#">Plans</a></li>
+      <li className={styles.navItem}><a className={styles.navLink} href="#"><Button type={"secondary"}>Sign Up</Button></a></li>
+    </ul>
+  )
+
```
Comment (HeibaiL) [inline src/layouts/Header/Header.tsx]: remove empty space

File: src/layouts/Header/Header.tsx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+//styles
+import styles from "@/styles/components/header.module.scss"
+
+const Header: FC = () => {
+  const [modalOpen, setModalOpen] = useState(false)
+  const { isMobile } = useWindowDimension();
+
+  const renderNavigationList = () => (
+    <ul className={styles.navList}>
```
Comment (HeibaiL) [inline src/layouts/Header/Header.tsx]: please, create an array NAVIGATION_LIST = [{title: "about", url:"#"}, {title: "Plans", url:"#"},  {title: "Plans", url:"#", hasButton: true}] and map it to render this list

File: src/layouts/Header/Header.tsx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+          <div className={styles.logo}>
+            <LogoImg/>
+            <p >YourCall</p>
+          </div>
+          <div className={classes(styles.navWrapper, { [styles.hidden]: isMobile })}>
+            {renderNavigationList()}
+          </div>
+          <div className={classes(styles.hamburger, { [styles.hidden]: !isMobile })}>
+            <div
+              onClick={ () => handleModalClick() }
```
Comment (HeibaiL) [inline src/layouts/Header/Header.tsx]: remove useless anonymus callback. use onClick={handleModalClick} instead

File: src/layouts/Header/Header.tsx [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+                <span></span>
+                <span></span>
+              </div>
+              <div id={styles.cross}>
+                <span></span>
+                <span></span>
+              </div>
+            </div>
+          </div>
+          <AntdDrawer width={"100vw"} closeIcon={null} className={styles.drawer} open={modalOpen} onClose={handleModalClick}>
```
Comment (HeibaiL) [inline src/layouts/Header/Header.tsx]: use new line for every property

File: src/layouts/Layout.tsx [hunk 10]
```diff
@@ -0,0 +1,18 @@
+import React, { FC } from "react";
+
+//components
+import Header from "./Header";
+
```
Comment (HeibaiL) [inline src/layouts/Layout.tsx:5]: add one additional space

File: src/pages/index.tsx [hunk 11]
```diff
@@ -18,111 +23,96 @@ export default function Home() {
...
-              Instantly deploy your Next.js site to a shareable URL
-              with&nbsp;Vercel.
-            </p>
-          </a>
-        </div>
       </main>
     </>
   )
 }
+
```
Comment (HeibaiL) [inline src/pages/index.tsx:112]: remove commented code

File: src/styles/components/button.module.scss [hunk 12]
```diff
@@ -1,11 +1,34 @@
 .button {
-        font-size: 20px
+  font-size: 20px;
+  font-family: 'Rubik', sans-serif;
```
Comment (HeibaiL) [inline src/styles/components/button.module.scss]: use variable for font family

File: src/styles/components/header.module.scss [hunk 13]
```diff
@@ -0,0 +1,203 @@
+.header {
+  z-index: 1001;
+  height: 110px;
+  width: 100%;
+  font-family: 'Rubik', sans-serif;
```
Comment (HeibaiL) [inline src/styles/components/header.module.scss]: same here

File: src/styles/pages/home.module.scss [hunk 14]
```diff
@@ -0,0 +1,256 @@
+main{
+  font-family: var(--main-font);
+}
+
+.preview {
+  padding: 55px 0;
+  margin-top: 110px;
+
```
Comment (HeibaiL) [inline src/styles/pages/home.module.scss:8]: remove one space

File: src/styles/pages/home.module.scss [hunk 15]
```diff
@@ -0,0 +1,256 @@
...
+    }
+
+    p {
+      margin: 50px 0;
+      font-weight: 400;
+      font-size: 23px;
+      line-height: 120%;
+    }
+
+    .previewImg {
```
Comment (HeibaiL) [inline src/styles/pages/home.module.scss]: remove

File: src/styles/pages/home.module.scss [hunk 16]
```diff
@@ -0,0 +1,256 @@
...
+        line-height: 120%;
+      }
+
+      .previewImg {
+        transform: scale(0.6);
+      }
+    }
+  }
+}
+
```
Comment (HeibaiL) [inline src/styles/pages/home.module.scss:75]: remove one space

File: src/styles/pages/home.module.scss [hunk 17]
```diff
@@ -0,0 +1,256 @@
...
+      }
+
+      p {
+        font-size: 18px;
+      }
+    }
+  }
+}
+
+
```
Comment (HeibaiL) [inline src/styles/pages/home.module.scss:130]: remove one space

File: src/styles/pages/home.module.scss [hunk 18]
```diff
@@ -0,0 +1,256 @@
...
+      }
+    }
+  }
+
+  @media (max-width: 850px) {
+    .featuresContainer {
+
+      .featuresList {
+        flex-direction: column;
+
```
Comment (HeibaiL) [inline src/styles/pages/home.module.scss]: remove one space

**Review by HeibaiL — CHANGES_REQUESTED** [review]

File: .storybook/preview.js (hunk 1 above)
Comment (Admiral-kalina) [reply .storybook/preview.js]: done

File: src/__stories__/Home.stories.tsx (hunk 2 above)
Comment (Admiral-kalina) [reply src/__stories__/Home.stories.tsx:6]: done

File: src/styles/pages/home.module.scss (hunk 14 above)
Comment (Admiral-kalina) [reply src/styles/pages/home.module.scss:8]: done

File: src/components/Header/components/Drawer.tsx [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+    <AntdDrawer width={"100vw"} closeIcon={null} className={styles.drawer} open={modalOpen}
+    >
+      <div className={classes(styles.navWrapper, styles.drawerContent)}>
+        {drawerContent}
+      </div>
+    </AntdDrawer>
+  );
+};
+
+export default Drawer;
```
Comment (HeibaiL) [inline src/components/Header/components/Drawer.tsx]: remove this file as it's useless

**Review by HeibaiL — APPROVED** [review]

**Pull request merged**